File: Elist/Elist/Models/Tasks.cs
namespace EList.Models
{
    public class Tasks
{
    public Guid task_id { get; set; }
    public string task_name { get; set; } = string.Empty;
    public string task_description { get; set; } = string.Empty;



    public DateTime due_date { get; set; }

    public bool task_isactive { get; set; }

}
}

File: Mobile App/elist/app/src/main/java/com/example/elist/model/Tasks.kt
package com.example.elist.model

import com.google.gson.annotations.SerializedName
data class Tasks(
    @SerializedName("task_id")
    val taskId: String = "",
    @SerializedName("task_name")
    val taskName: String = "",
    @SerializedName("task_description")
    val taskDescription: String = "",
    @SerializedName("due_date")
    val dueDate: String = "",
    val isactive: Boolean = false
)